src/Features/CSharp/CSharpFeaturesResources.resx

```diff
@@ -228,6 +228,9 @@
   <data name="Deprecated" xml:space="preserve">
     <value>deprecated</value>
```

Comment:
Actually (per our offline discussion) the Deprecated string isn't used anywhere after this change, so you can just remove it.

@Pilchie @DustinCampbell Do we want to *also* support [Deprecated] for the actual [DeprecatedAttribute](https://msdn.microsoft.com/library/windows/apps/windows.foundation.metadata.deprecatedattribute.aspx)? QuickInfo for DeprecatedAttribute methods does not show the symbol display prefix [Deprecated] or [Obsolete] but does show an add-on `'method' is obsolete: 'message'`.